.gitignore

```diff
@@ -78,6 +78,9 @@ target/
 # Jupyter Notebook
 .ipynb_checkpoints
 
+# IDEA files
+/.idea
```

Comment:
```suggestion
.idea/
```

the .idea is still showing up (you can see for self in the commits. I think you have the syntax backwards.

by the way, you can directly commit this suggestion to save time.

Reply:
Hmm yeah I was seeing both on Stack Overflow. Is it working now?

Reply:
I still see the files under “Files changed”. You might need to explicitly delete them and push that